src/main/asciidoc/_chapters/security.adoc

```diff
@@ -71,6 +71,19 @@ See Nick Dimiduk's contribution on this link:http://stackoverflow.com/questions/
 If you know how to fix this without opening a second port for HTTPS, patches are appreciated.
 ====
 
+[[hbase.ui.cache]]
+=== Disable cache in HBase UI
+
```

Comment:
The wiki seems to be broken with this change, please check

Reply:
Resolved in latest commit